pyproject.toml

```diff
@@ -0,0 +1,86 @@
+[build-system]
+requires = ["setuptools >= 38.6", "pip >= 10"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pysatSpaceWeather"
+version = "0.0.10"
+description = 'pysat support for Space Weather Indices'
+readme = "README.md"
+requires-python = ">=3.6"
+license = {file = "LICENSE"}
+authors = [
+    {name = "Angeline Burrell", email = "pysat.developers@gmail.com"}
+]
+classifiers = [
+  "Development Status :: 3 - Alpha",
```

Comment:
Is this still Alpha?

Reply:
Eh, I made some RC changes throughout.  The next release will go from alpha to beta and I jumped the gun a little bit.